src/base/perf.hpp

```diff
@@ -140,14 +139,23 @@ namespace cubperf
   //
   // see how to use in file description comment
   //
-  class statset_definition
+  class statset_definition final
   {
     public:
-      // no default constructor
+      using stat_definition_init_list_t = std::initializer_list<stat_definition>;
```

Comment:
If it is part of statset_definition, repeating the same suffix is not necessary.
```suggestion
      using init_list_t = std::initializer_list<stat_definition>;
```